cli/internal/run/run_test.go

```diff
@@ -306,14 +306,14 @@ func Test_dontSquashTasks(t *testing.T) {
 
 	pipeline := map[string]fs.TaskDefinition{
 		"build": {
-			Outputs:          []string{},
+			Outputs:          fs.TaskOutputs{Inclusions: []string{}, Exclusions: []string{}},
```

Comment:
Go idiom: typically we try to make the "zero" value valid. So, `fs.TaskOutputs{}` should work. Nearly all code expecting a slice _should_ work with `nil`, if there's some piece that doesn't, it's probably worth fixing.

Reply:
Gotta turn off Rust brain haha. Makes me want to avoid `nil`